bin/oref0-setup.sh

```diff
@@ -1025,8 +1030,6 @@ if prompt_yn "" N; then
         else
             # radio_locale requires openaps 0.2.0-dev or later
             openaps device add pump mmeowlink subg_rfspy $ttyport $serial $radio_locale || die "Can't add pump"
```

Comment:
These lines normally add the pump stanza for MDT CGM users. We only do so manually for non-Enlite users because they’re using the Go based pump comms and not the old openaps toolkit.

Reply:
i'm just removing support for ww_ti_usb_reset and removing all the `oref0_init_pump_comms.py` references. I don't understand what you mean with the comment above